android/build.gradle

```diff
@@ -62,4 +62,8 @@ dependencies {
     implementation "com.adyen.checkout:drop-in:4.12.1"
     implementation 'com.google.android.material:material:1.7.0'
     implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.4.0"
+
+    //Tests
+    testImplementation "junit:junit:4.13.2"
```

Comment:
I'd advise you to use junit 5 and mockito Kotlin as they can make your tests more readable and come with some nice features

Reply:
For now, it serves a purpose. On v2 I'll apply the latest junit and mockito-kotlin.